Assets/Scripts/Explosion.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Assertions;
+
+[RequireComponent(typeof(SphereCollider))]
+public class Explosion : MonoBehaviour
+{
+	public const float MinExplosionRange = 0f;
+	public const float MaxExplosionRange = 20f;
+	public const float MinVisualExpansionRange = 0f;
+	public const float MaxVisualExpansionRange = 1f;
+	public const float MinColliderExpansionRange = 0f;
+	public const float MaxColliderExpansionRange = 0.1f;
+
+	[Range(MinExplosionRange, MaxExplosionRange)]
+    [SerializeField]
+    private float maxExplosionRadius;
+	[Range(MinVisualExpansionRange, MaxVisualExpansionRange)]
+    [SerializeField]
+    private float visualExpansionPerFrame;
+	[Range(MinColliderExpansionRange, MaxColliderExpansionRange)]
+    [SerializeField]
+    private float colliderExpansionPerFrame;
+
+    private SphereCollider sphereCollider;
+    private Vector3 expansionVector;
+
+    void Start()
```

Comment:
You haven't done assert checks for `visualExpansionsPerFrame` or `colliderExpansionPerFrame`